### DISCUSSION TITLE
Enabling screensaver when gaming?

### DISCUSSION THREAD

**Discussion by MadLimes** [rocksdanister/lively#1804] [Q&A]
Hello, I have had lively for a couple of months now and I am frustrated that the screen saver does not activate when I am gaming. I want the screensaver to turn on even if I have a game open. Is there anyway I can do this?

**Comment by rocksdanister** [comment]
You could write a ahk script that does this:
https://github.com/rocksdanister/lively/wiki/Command-Line-Controls
